app/views/govuk_publishing_components/components/_previous_and_next_navigation.html.erb

```diff
@@ -2,12 +2,16 @@
   add_gem_component_stylesheet("previous-and-next-navigation")
 
   if local_assigns.include?(:next_page) || local_assigns.include?(:previous_page)
```

Comment:
Unrelated: seems slightly strange to have an `if` statement begin here as part of a larger code block, was initially a bit stumped trying to understand this - I'd expect it to be a single erb statement i.e. `<% if ... %>`